conmon/conmon.c

```diff
@@ -166,6 +167,7 @@ static GOptionEntry opt_entries[] = {
 	{"replace-listen-pid", 0, 0, G_OPTION_ARG_NONE, &opt_replace_listen_pid, "Replace listen pid if set for oci-runtime pid", NULL},
 	{"bundle", 'b', 0, G_OPTION_ARG_STRING, &opt_bundle_path, "Bundle path", NULL},
 	{"pidfile", 'p', 0, G_OPTION_ARG_STRING, &opt_pid_file, "PID file", NULL},
```

Comment:
Should we deprecate this one and namespace this option as well (`--container-pidfile`?)?  Without that, I think folks will have to dig around to figure out how `--pidfile` and `--conmon-pidfile` differed, especially since your current help text is “PID file” for both ;).

Reply:
Yes I think this one should be container-pidfile and the first one be pidfile.  Perhaps use the short -P for pidfile and -p for container-pidfile